sdk/search/azure-search/azure/search/_index/__init__.py

```diff
@@ -0,0 +1,14 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from ._credential import SearchApiKeyCredential
+from ._index_batch import IndexBatch
+from ._search_index_client import SearchIndexClient
+from ._queries import (  # pylint: disable=unused-import
+    AutocompleteQuery,
+    SearchQuery,
+    SuggestQuery,
+)
+
+__all__ = ("IndexBatch", "SearchApiKeyCredential", "SearchIndexClient", "SearchQuery")
```

Comment:
Do you need a `__all__` from an internal module? Is anyone actually using this?

Reply:
removed

Reply:
removed